README.md

```diff
@@ -1,76 +1,84 @@
 <summary>🌺 Macchiato</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta/main/assets/macchiato.webp"/>
+<img src="assets/macchiato.webp"/>
 </details>
 <details>
-<summary>🌿 Mocha</summary>
-<img src="https://raw.githubusercontent.com/riivx/vendetta//main/assets/mocha.webp"/>
+<summary>🌿 Mocha</summary> 
+<img src="assets/mocha.webp"/>
 </details>
 
-## Usage
 
->[!WARNING]
->Vendetta was **discontinued** on February 7. 2024. Use [Bunny](https://github.com/pyoncord/Bunny) instead. It supports Vendetta themes
+## Usage
+> [!IMPORTANT]
+> This is the home for the shared theme format that [Enmity](https://github.com/enmity-mod/enmity), [Bunny](https://github.com/pyoncord/Bunny) & [† Vendetta](https://github.com/vendetta-mod/Vendetta) use.
 
 1. Choose a flavor, and copy its link below:
 	- **Latte**: https://raw.githubusercontent.com/catppuccin/vendetta/main/latte.json
 	- **Frappe**: https://raw.githubusercontent.com/catppuccin/vendetta/main/frappe.json
 	- **Macchiato**: https://raw.githubusercontent.com/catppuccin/vendetta/main/macchiato.json
 	- **Mocha**:  https://raw.githubusercontent.com/catppuccin/vendetta/main/mocha.json
-2. Head to `You Tab (aka Settings)` > `Vendetta` > `Themes`
+2. Head to `Settings` > `Bunny` or `Enmity` > `Themes`
 3. Paste the copied link and press `Install`
 4. Select your theme and you're ready to go!
 
-> [!IMPORTANT]
+> [!NOTE]
 > Light mode can be used **only with Latte theme**.
 
 
-## Icon packs
-
-1. Copy `https://vendetta.nexpid.xyz/themes-plus`
-2. Head to `You Tab (aka Settings)` > `Vendetta` > `Plugins`
+## Themes+
+Themes+ is required to theme some hardcoded elements (applied when themes+ is loaded) as well as iconpacks.
+#### Iconpacks
+1. Copy `https://bunny.nexpid.xyz/themes-plus`
+2. Head to `Settings` > `Bunny` > `Plugins`
 3. Paste the copied link and press `Install`
-4. Go back to `Vendetta` > `plugins` > `themes+ settings`
-5. Double tap `THEMES+ IS ACTIVE` > Choose icon pack
+4. Go back to `Bunny` > `Plugins` > `themes+ settings`
+5. Click `Configuration` > `Mode` > `Manual` > Choose icon pack
 6. Hit reload
+> [!IMPORTANT]
+> If the icon pack doesn't apply after discord loads, then switch to a different server and back.
+>
+> If you are using Enmity, install [VendettaCommpat](https://discord.com/channels/950850315601711176/1019645460589318165/1211473409154744350) plugin first. After doing so, follow this [guide](https://discord.com/channels/950850315601711176/1239393517151846412) as it loads Vendetta by default which is broken past Discord version 223
 
+## Fonts
+### Bunny
+1. Head to `Settings` > `Bunny` > `Fonts` > `add button`
+2. Follow shown options
 > [!NOTE]
-> If the icon pack doesn't apply after discord loads, then switch to a different server and back.
+> The font pack shown in the screenshots is [Flow Circular](https://gist.githubusercontent.com/Covkie/d4124cfc4df15e7790167ac88e820e0d/raw/02f0c7124f233ee236f0682d464c7cb22c2ea6e9/FlowCircular-font-snippet.json)
```

Comment:
preview of the theme has no text at all, so we should probably move this somewhere

Reply:
my bad, then we're keeping it (don't know who will use that, but still)